tests/task_router/test_workflow_config.py

```diff
@@ -78,6 +78,108 @@ def test_from_json2(self):
             self.assertEqual(2, len(config.task_routing.filters))
             self.assertEqual(4, len(config.task_routing.default_filter))
 
+        def test_from_json_filter_friendly_name(self):
+
+            data = {
+                       'task_routing':
+                       {
+                           'filters': [
+                               {
+                                   'expression': 'type == "sales"',
+                                   'friendly_name': 'Sales',
```

Comment:
this test doesn't validate that we can convert `filter_friendly_name` into `friendly_name` because the input data uses `friendly_name`